docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,121 +4,113 @@
 <titleabbrev>Encrypting communications</titleabbrev>
 ++++
 
-{kib} supports Transport Layer Security (TLS/SSL) encryption for client 
-requests. 
-//TBD: It is unclear what "client requests" are in this context. Is it just 
-// communication between the browser and the Kibana server or are we talking 
-// about other types of clients connecting to the Kibana server?
-
-If you are using {security} or a proxy that provides an HTTPS endpoint for {es}, 
-you can configure {kib} to access {es} via HTTPS. Thus, communications between 
-{kib} and {es} are also encrypted. 
-
-. Configure {kib} to encrypt communications between the browser and the {kib}
-server:
+{kib} supports Transport Layer Security (TLS/SSL) encryption for all forms of data-in-transit. Browsers will send traffic to {kib}, and
```

Comment:
Switching to present tense instead of future:
```suggestion
{kib} supports Transport Layer Security (TLS/SSL) encryption for all forms of data-in-transit. Browsers send traffic to {kib} and
```